src/common/exif.cc

```diff
@@ -1065,7 +1065,10 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     if(FIND_EXIF_TAG("Exif.Photo.UserComment"))
     {
       std::string str = pos->print(&exifData);
-      dt_metadata_set_import(img->id, "Xmp.dc.description", str.c_str());
+      Exiv2::CommentValue value(str);
+      const char *comment = value.comment().c_str();
```

Comment:
This line causes LLVM to warn:

`src/src/common/exif.cc:1069:29: fatal error: object backing the pointer will be destroyed at the end of the full-expression [-Wdangling-gsl]`